.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "seneca"
+  "extends": "seneca",
+  "rules": {
```

Comment:
( authToken? )

Disabling still not accepted :)
We try to make uniform code across modules, disabling rules would cancel the purpose of having common rules. Let me know if you need me to help.